packages/js/src/lib/display.ts

```diff
@@ -1,21 +1,23 @@
+import { TDisplay, TDisplayInput } from "@formbricks/types/v1/displays";
 import type { DisplayCreateRequest, JsConfig, Response } from "../../../types/js";
 import { NetworkError, Result, err, ok, okVoid } from "./errors";
 
 export const createDisplay = async (
-  displayCreateRequest: DisplayCreateRequest,
+  displayCreateRequest: TDisplayInput,
   config: JsConfig
-): Promise<Result<Response, NetworkError>> => {
-  const url = `${config.apiHost}/api/v1/client/environments/${config.environmentId}/displays`;
+): Promise<Result<TDisplay, NetworkError>> => {
+  // const url = `${config.apiHost}/api/v1/client/environments/${config.environmentId}/displays`;
+  const url = `${config.apiHost}/api/v1/client/displays`;
 
   const res = await fetch(url, {
     method: "POST",
     headers: { "Content-Type": "application/json" },
     body: JSON.stringify(displayCreateRequest),
   });
 
-  if (!res.ok) {
-    const jsonRes = await res.json();
+  const jsonRes = await res.json();
```

Comment:
it's better to move this call below the `if (!res.ok) {` check because it's only necessary when the response was successful and can potentially fail for non-json payload.